documentation/stories/servicePath/formCreation.jsx

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types';
+import Grid from '@mui/material/Grid';
+import ResourceForm from '../../../src/components/resource/resourceForm';
+import { BrowserRouter } from 'react-router-dom';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { orange } from '@mui/material/colors';
+import '../../../src/i18n';
+import React from 'react';
+import { SnackbarProvider } from 'notistack';
+
+/**
+ * Primary UI component for user interaction
+ */
+const theme = createTheme({
+  status: {
+    danger: orange[500]
+  }
+});
+
+export const ResourceTypeCreation = ({ title, close, action, token, tokenData, env, getTheResources, thisTenant }) => {
```

Comment:
why this files are under servicePath?

Reply:
> why this files are under servicePath?

It's the form for the creation of the service path for the documentation canvas

Reply:
it reports `ResourceTypeCreation `

Reply:
> it reports `ResourceTypeCreation `

ok i get it! Misspelling...

Reply:
not sure it's misspelling or what, simply the folder should probably not be `documentation/stories/servicePath` but something like `documentation/stories/resourceType` (also considering that there is a folder name `service` where docs for servicePath seems to be contained)